src/unsupported/NodeHealthCheck.py

```diff
@@ -86,7 +86,7 @@
 
 
 # Define the version
-__version__ = '0.0.2'
+__version__ = '0.0.3'
```

Comment:
Why is __version__ defined here? It's not used anywhere.

Reply:
It must have been originally meant to keep versioning info and so I updated it.

Reply:
But is it needed? I think it should be removed if it's not used for anything or it will continue to confuse people.

Reply:
It is not needed since we use github for our repository and I have removed __version__ .